Rubberduck.Parsing/VBA/ParseCoordinator.cs

```diff
@@ -839,22 +839,12 @@ private void SyncComReferences(IReadOnlyList<IVBProject> projects, CancellationT
             var unmapped = new ConcurrentBag<IReference>();
 
             var referencesToLoad = GetReferencesToLoadAndSaveReferencePriority(projects);
-                            
-            State.OnStatusMessageUpdate(ParserState.LoadingReference.ToString());
```

Comment:
And I just realized why that "Loading references" caption didn't follow the localization settings...